examples/taskworkflow.go

```diff
@@ -80,8 +92,24 @@ func taskWorkflow(containerIP string) error {
 
 	task, err := container.NewTask(ctx,
 		cio.NewCreator(cio.WithStdio),
-		withFirecrackerConfig(containerIP, "AA:FC:00:00:00:01", "tap0"),
-	)
+		func(ctx context.Context, _ *containerd.Client, ti *containerd.TaskInfo) error {
```

Comment:
How come you put this inline instead of leaving it as a separate function?

Reply:
I didn't really feel the need for a separate function for this as it'd just be passing a number of parameters that are already here everywhere. Will revert to a named method if there are more complicated use-cases to consider.